src/main/java/org/mockito/internal/util/Checks.java

```diff
@@ -24,4 +24,11 @@
         }
         return iterable;
     }
+
+    public static <T> T check(boolean assertion, T arg, String checkedValue) {
```

Comment:
Ha good it was added then removed after refactoring, maybe it's the reason why the code coverage is less.